swift/Tests/Blockchains/BitsharesTests.swift

```diff
@@ -0,0 +1,135 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import XCTest
+import TrustWalletCore
+
+class BitsharesTests: XCTestCase {
+    override func setUp() {
+        continueAfterFailure = false
+    }
+
+    func testAddresses() {
+        XCTAssertTrue(CoinType.bitshares.validate(address: "BTS52K5kMmwiRyNQYAf7ymCMz6hieE8siyrqNt1t57ac9hvBrRdaa"))
+        for addr in [
+            "52K5kMmwiRyNQYAf7ymCMz6hieE8siyrqNt1t57ac9hvBrRdaa",
+            "BTS52K5kMmwiRyNQYAf7ymCMz6hieE8siyrqNt1t57ac9hvBrRd",
+            "BTS52K5kMmwiRyNQYAf7ymCMz6hieE8siyrqNt1t57ac9hvBrRdaaaa",
+            "BITS52K5kMmwiRyNQYAf7ymCMz6hieE8siyrqNt1t57ac9hvBrRdaa",
+            ] {
+            XCTAssertFalse(CoinType.steem.validate(address: addr))
+        }
+    }
+    
+    
+    let signingInput: BitsharesSigningInput = {
+        let asset = BitsharesAsset.with {
+            $0.amount = 511;
```

Comment:
you don't need to add `;` in swift

Reply:
We have fixed this and all other codacy swift issues.